include/gmsh.tcc

```diff
@@ -194,12 +194,111 @@ void GMSH<Tdim, Tvertices>::store_element_vertices() {
       << "The coordinates for vertices of each element have been stored\n";
 }
 
-//! Compute material points based on the centroid
+//! Compute material points for 2D quadrilateral
```

Comment:
The function name is not clear, what is it supposed to do?

Reply:
What do we mean by `compute`? what are we computing?

Reply:
I would use `generate` then